components/brave_rewards/resources/shared/components/newtab/rewards_card.tsx

```diff
@@ -35,7 +35,7 @@ const monthDayFormatter = new Intl.DateTimeFormat(undefined, {
 function renderDateRange () {
   const now = new Date()
   const start = new Date(now.getFullYear(), now.getMonth(), 1)
-  const end = new Date(start.getFullYear(), start.getMonth() + 1, -1)
+  const end = new Date(start.getFullYear(), start.getMonth() + 1, 0)
```

Comment:
Could we add a short comment to explain the trick we're using here (I'm assuming it's getting the "0th" day of the next month, which means the last day of the current month?)

Separately, will adding 1 to the month do the right thing in December?

Reply:
Good idea - comment added.

> Separately, will adding 1 to the month do the right thing in December?

It will:

```
> new Date(2021, 12, 0)
2021-12-31T05:00:00.000Z
```

From the ES spec:

```
7. Let mn be 𝔽(ℝ(m) modulo 12).
```

Reply:
Perfect!